core/src/test/java/org/testcontainers/images/builder/DockerignoreTest.java

```diff
@@ -0,0 +1,53 @@
+package org.testcontainers.images.builder;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+import static org.rnorth.visibleassertions.VisibleAssertions.fail;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+
+import com.github.dockerjava.api.exception.DockerClientException;
+
+public class DockerignoreTest {
+    
+    private static final Path INVALID_DOCKERIGNORE_PATH = Paths.get("src/test/resources/dockerfile-build-invalid");
+    
+    @Test
+    public void testInvalidDockerignore() throws Exception {
+        try {
+            new ImageFromDockerfile()
+                .withFileFromPath(".", INVALID_DOCKERIGNORE_PATH)
+                .withDockerfile(new File("src/test/resources/dockerfile-build-invalid/Dockerfile"))
+                .get();
+            fail("Should not be able to build an image with an invalid .dockerignore file");
+        } catch(DockerClientException e) {
+            if (!e.getMessage().contains("Invalid pattern"))
+                throw e;
+        }
+    }
+    
+    @Test
+    public void testValidDockerignore() throws Exception {
+        try (final GenericContainer container = new GenericContainer(new ImageFromDockerfile()
```

Comment:
nit1: please extract the image definition

nit2: could you please add a line break after `try (` and also before `) {`? I know that we're a bit inconsistent, but usually it helps, especially with long fluent-style chains